tapalcatl_server/server.go

```diff
@@ -327,7 +332,7 @@ func main() {
 `)
 	f.StringVar(&listen, "listen", ":8080", "interface and port to listen on")
 	f.String("config", "", "Config file to read values from.")
-	f.StringVar(&healthcheck, "healthcheck", "", "A path to respond to with a blank 200 OK. Intended for use by load balancer health checks.")
+	f.StringVar(&healthcheck, "healthcheck", "", "A URL path for healthcheck. Intended for use by load balancer health checks.")
```

Comment:
This is the configuration to specify the URL path to attach the healthcheck listener to. The other configuration stuff was for where to look for a file in the `storage` to perform the check for upness.